apps/qubit/modules/settings/actions/pageElementsAction.class.php

```diff
@@ -63,5 +65,23 @@ public function execute($request)
 
       $this->defaultPageElementsForm->setDefault('toggleTitle', $toggleTitleSetting->__get('value', array('sourceCulture' => true)));
     }
+
+    $criteria = new Criteria;
+    $criteria->add(QubitSetting::NAME, 'toggleLanguageMenu');
+    if (1 == count($toggleLanguageMenuQuery = QubitSetting::get($criteria)))
```

Comment:
I'm not really a fan of how we're doing the ifs() here. QubitSetting::getOne and then checking against null would be preferable.